src/ert/config/ensemble_config.py

```diff
@@ -104,6 +104,7 @@ def __init__(
             self.addNode(gen_kw)
 
         for surface in _surface_list:
+            logger.info(f"Adding surface {surface.name} to ensemble config")
```

Comment:
Suggest moving this into `addNode` under:
`if isinstance(config_node, ParameterConfig):`

and then logging the type and the size. It will then cover all the parameters types, so we get a good overview of the total usage.